test/unittests/libp2p/eip-8.cpp

```diff
@@ -1,40 +1,21 @@
-/*
-    This file is part of cpp-ethereum.
-
-    cpp-ethereum is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-
-    cpp-ethereum is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License
-    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
-*/
-/** @file discovery.cpp
- * @author Felix Lange <fjl@twurst.com>
- * @date 2016
- * Forward-compatibility tests checking EIP-8 compliance.
- */
-
-#include <boost/test/unit_test.hpp>
-#include <libp2p/NodeTable.h>
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+/// @file
+/// Forward-compatibility tests checking EIP-8 compliance.
+
 #include <libp2p/Host.h>
+#include <libp2p/NodeTable.h>
 #include <libp2p/RLPXSocket.h>
 #include <libp2p/RLPxHandshake.h>
-#include <test/tools/libtesteth/TestOutputHelper.h>
+#include <gtest/gtest.h>
 
 using namespace std;
 using namespace dev;
-using namespace dev::test;
 using namespace dev::p2p;
 
-BOOST_FIXTURE_TEST_SUITE(eip8, TestOutputHelperFixture)
-
-BOOST_AUTO_TEST_CASE(test_discovery_packets)
+TEST(eip8, test_discovery_packets)
```

Comment:
I think we wanted to use camelCase for test names?

Reply:
Would be good at some point to convert names. And maybe add library prefixes.